src/library-authoring/data/api.mocks.ts

```diff
@@ -1091,6 +1137,7 @@ export async function mockGetCourseImports(libraryId: string): ReturnType<typeof
 mockGetCourseImports.libraryId = mockContentLibrary.libraryId;
 mockGetCourseImports.emptyLibraryId = mockContentLibrary.libraryId2;
 mockGetCourseImports.succeedImport = {
+  taskUuid: '1',
```

Comment:
Is taskUuid an integer in a string (seems wrong), or should it be a UUID like you get from https://www.uuidgenerator.net/ ? I think it's better to use an accurate example in the mock data.

```suggestion
  taskUuid: '2d35e36b-1234-1234-1234-123456789000',
```